tests/workflow.spec.ts

```diff
@@ -11,12 +11,16 @@ import { LONG, SHORT, STAGE, BROWSER } from "../base/tags";
 import { CalendarPage } from "../pages/Cards/Calendar.page";
 import { DirectoryPage } from "../pages/Cards/Directory.page";
 import { createTeacherAndLogin } from "../base/api/testHelperPerson.page";
+import { UserInfo} from "../base/api/testHelper.page.ts";
+import { deletePersonByUsername } from "../base/testHelperDeleteTestdata.ts";
 
 const PW: string | undefined = process.env.PW;
 const ADMIN: string | undefined = process.env.USER;
 const FRONTEND_URL: string | undefined = process.env.FRONTEND_URL || "";
 const ENV: string | undefined = process.env.ENV;
 
+let username: string[] = []; // Im afterEach Block werden alle Testdaten gelöscht
```

Comment:
```suggestion
let usernames: string[] = []; // Im afterEach Block werden alle Testdaten gelöscht
```
weil es ein array von strings ist

Reply:
Die Verwendung der Variable muss auch angepasst werden.

Reply:
done